internal/table/console.go

```diff
@@ -69,9 +69,9 @@ WriteHeader outputs header for the table with the form:
 func (t *TabularWriter) WriteHeader(cells ...interface{}) error {
 	t.initSize(cells)
 	// colWidth = (totalWidth - numOfSeparators) / numOfColumns
-	colWidth := (t.columnCount - (len(cells)+1)*2) / len(cells)
-	unusedCellCount := (t.columnCount - (len(cells)+1)*2) % len(cells)
-	headerBorder := fmt.Sprintf("+%s+\n", strings.Repeat("-", t.columnCount-unusedCellCount-2))
+	colWidth := (t.columnCount - (len(cells) + 2)) / len(cells)
```

Comment:
Same question about negative `colWidth` here.

Reply:
Btw, if `-2` is something we have to use, it would be better to make it a constant to convey its purpose better. Probably for the `+1` below, if it's something special.

Reply:
Same comment for other `-2`s. It would be great to have a single constant if they are the same thing.

Reply:
I did not have a problem with negative colWidth, trying to minimize the terminal as much as I can.

I have moved -2 to constants here https://github.com/hazelcast/hazelcast-commandline-client/pull/61/commits/f6cdf08d70a3994833449303fb682e7f7b717d6d

Reply:
OK, I thought that was leftover of a previous code. How about uncommenting that line and set `numOfSeparators` (or maybe `sepCnt` if you'd like to make it shorter), etc.? I think that will make the code more understandable.

Reply:
Great idea, refactored on https://github.com/hazelcast/hazelcast-commandline-client/pull/61/commits/4cff1e6baac2c26537a010cf79a33bc6cf07c476